protocol/streaming/full_node_streaming_manager.go

```diff
@@ -459,6 +596,42 @@ func (sm *FullNodeStreamingManagerImpl) AddUpdatesToCache(
 	sm.EmitMetrics()
 }
 
+// AddSubaccountUpdatesToCache adds a series of updates to the full node streaming cache.
+// Subaccount ids are the subaccount id each update is relevant to.
+func (sm *FullNodeStreamingManagerImpl) AddSubaccountUpdatesToCache(
+	updates []clobtypes.StreamUpdate,
+	subaccountIds []*satypes.SubaccountId,
+	numUpdatesToAdd uint32,
```

Comment:
don't need this field, can just use `len(updates)`

Reply:
done